scripts/lv_conf_internal_gen.py

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env python3
 
 '''
-Generates a checker file for lv_conf.h from lv_conf_template.h define all the not defined values
+Generates a checker file for lv_conf_internal.h from lv_conf_template.h define all the not defined values
```

Comment:
It's really for `lv_conf.h` but generates `lv_conf_internal.h`. So I think the comment was correct.

Reply:
The source is lv_conf_template.h, and the target is lv_conf_internal.h, not lv_conf.h.

Reply:
The goal of `lv_conf_internal.h` is to check `lv_conf.h`, so technically the original comment was correct, however, I would suggest we just change it to this to be explicit:

```suggestion
Generates lv_conf_internal.h from lv_conf_template.h to provide default values
```

Reply:
Done